services/catalog/src/simcore_service_catalog/db/repositories/services.py

```diff
@@ -203,13 +203,16 @@ async def create_service(
                 raise ValueError(
                     f"{access_rights} does not correspond to service {new_service.key}:{new_service.version}"
                 )
-
+        # Set the deprecation datetime to None (will be converted top sql's null) if not given
+        new_service_dict = new_service.dict(by_alias=True)
+        if "deprecated" not in new_service_dict:
+            new_service_dict["deprecated"] = None
         async with self.db_engine.begin() as conn:
             # NOTE: this ensure proper rollback in case of issue
-            result = await conn.execute(
+            result = await conn.execute(  ## Here, maybe add default deprecation date?
```

Comment:
If this is not a todo:
1. Why are you hesitating in adding it?
1. What is a default deprecation date, can you explain?